host/src/app.c

```diff
@@ -721,6 +731,15 @@ int app_main(int argc, char * argv[])
         LG_UNLOCK(app.pointerLock);
       }
 
+      const uint64_t now = microtime();
+      const uint64_t delta = now - previousFrameTime;
+      if (delta < throttleUs) {
+        usleep(throttleUs - delta);
+        previousFrameTime = microtime();
+      } else {
```

Comment:
ditto here, format for this project is:
```c
}
else
{
```

for single line if statements, no braces are needed unless the code is hard to follow without them.